.readthedocs.yml

```diff
@@ -12,7 +12,6 @@ build:
     pre_install:
       - git update-index --assume-unchanged docs/requirements.txt docs/source/conf.py
     pre_build:
-      - python -m pip install --no-deps --no-cache-dir --editable .
```

Comment:
You need that so it can find conda

Reply:
nope, no longer needed since we're using `sphinx-autoapi` (along with modifying the search path that we do at the top of `conf.py`)

Reply:
The problem is this, effectively it can't import conda without it https://continuumio-conda-docs--12798.com.readthedocs.build/projects/conda/en/12798/configuration.html Alternatively we could add the repo root to `PYTHONPATH` in `conf.py`

Reply:
grr i keep forgetting the configuration docs 🤦🏼‍♂️

as for `PYTHONPATH`, we already add the root directory to `sys.path` so it should (?) work but i guess not, i'll just revert this for simplicity

Reply:
Sorry :( I haven't debugged why this is failing. My guess is that it could be fixed if/when we do docs build infra updates.